2022-03-31 18:47:02,891 INFO    MainThread:29389 [wandb_setup.py:_flush():75] Loading settings from /home/ltorres/.config/wandb/settings
2022-03-31 18:47:02,892 INFO    MainThread:29389 [wandb_setup.py:_flush():75] Loading settings from /home/ltorres/leo/tesis/cloud-classification/code/wandb/settings
2022-03-31 18:47:02,892 INFO    MainThread:29389 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-31 18:47:02,892 INFO    MainThread:29389 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-31 18:47:02,892 INFO    MainThread:29389 [wandb_init.py:_log_setup():405] Logging user logs to /home/ltorres/leo/tesis/cloud-classification/code/wandb/run-20220331_184702-abq1xa25/logs/debug.log
2022-03-31 18:47:02,892 INFO    MainThread:29389 [wandb_init.py:_log_setup():406] Logging internal logs to /home/ltorres/leo/tesis/cloud-classification/code/wandb/run-20220331_184702-abq1xa25/logs/debug-internal.log
2022-03-31 18:47:02,892 INFO    MainThread:29389 [wandb_init.py:init():439] calling init triggers
2022-03-31 18:47:02,892 INFO    MainThread:29389 [wandb_init.py:init():442] wandb.init called with sweep_config: {}
config: {'learning_rate': 0.0001, 'architecture': 'GATConvGNN', 'loss': 'LogeLoss', 'optim': 'SGD', 'dataset': 'GCD', 'epochs': 100}
2022-03-31 18:47:02,892 INFO    MainThread:29389 [wandb_init.py:init():458] re-initializing run, found existing run on stack: 1wv8lsjj
2022-03-31 18:47:02,893 INFO    MainThread:29389 [wandb_run.py:finish():1539] finishing run leonardtd/cloud classification/1wv8lsjj
2022-03-31 18:47:02,893 INFO    MainThread:29389 [jupyter.py:save_history():429] not saving jupyter history
2022-03-31 18:47:02,894 INFO    MainThread:29389 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-31 18:47:02,894 INFO    MainThread:29389 [wandb_init.py:_jupyter_teardown():337] cleaning up jupyter logic
2022-03-31 18:47:02,894 INFO    MainThread:29389 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-03-31 18:47:02,895 INFO    MainThread:29389 [wandb_run.py:_restore():1674] restore
2022-03-31 18:47:03,619 INFO    MainThread:29389 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 909
  total_bytes: 909
}

2022-03-31 18:47:03,930 INFO    MainThread:29389 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 909
  total_bytes: 909
}

2022-03-31 18:47:04,351 INFO    MainThread:29389 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 909
  total_bytes: 3956
}

2022-03-31 18:47:04,454 INFO    MainThread:29389 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 909
  total_bytes: 10399
}

2022-03-31 18:47:04,557 INFO    MainThread:29389 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 909
  total_bytes: 10399
}

2022-03-31 18:47:04,659 INFO    MainThread:29389 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 909
  total_bytes: 10399
}

2022-03-31 18:47:04,762 INFO    MainThread:29389 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 909
  total_bytes: 10399
}

2022-03-31 18:47:04,864 INFO    MainThread:29389 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10399
  total_bytes: 10399
}

2022-03-31 18:47:04,967 INFO    MainThread:29389 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10399
  total_bytes: 10399
}

2022-03-31 18:47:05,069 INFO    MainThread:29389 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10399
  total_bytes: 10399
}

2022-03-31 18:47:05,172 INFO    MainThread:29389 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10399
  total_bytes: 10399
}

2022-03-31 18:47:05,274 INFO    MainThread:29389 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10399
  total_bytes: 10399
}

2022-03-31 18:47:05,377 INFO    MainThread:29389 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10399
  total_bytes: 10399
}

2022-03-31 18:47:05,616 INFO    MainThread:29389 [wandb_run.py:_on_finish():1831] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10399
  total_bytes: 10399
}

2022-03-31 18:47:06,237 INFO    MainThread:29389 [wandb_run.py:_on_finish():1831] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 10399
  total_bytes: 10399
}
local_info {
}

2022-03-31 18:47:07,381 INFO    MainThread:29389 [wandb_run.py:_footer_history_summary_info():2865] rendering history
2022-03-31 18:47:07,381 INFO    MainThread:29389 [wandb_run.py:_footer_history_summary_info():2894] rendering summary
2022-03-31 18:47:07,383 INFO    MainThread:29389 [wandb_run.py:_footer_sync_info():2822] logging synced files
2022-03-31 18:47:07,389 INFO    MainThread:29389 [wandb_init.py:init():492] starting backend
2022-03-31 18:47:07,389 INFO    MainThread:29389 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-31 18:47:07,391 INFO    MainThread:29389 [backend.py:ensure_launched():219] starting backend process...
2022-03-31 18:47:07,422 INFO    MainThread:29389 [backend.py:ensure_launched():224] started backend process with pid: 29749
2022-03-31 18:47:07,424 INFO    MainThread:29389 [wandb_init.py:init():501] backend started and connected
2022-03-31 18:47:07,441 INFO    MainThread:29389 [wandb_run.py:_label_probe_notebook():947] probe notebook
2022-03-31 18:47:07,442 INFO    MainThread:29389 [wandb_run.py:_label_probe_notebook():957] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-03-31 18:47:07,442 INFO    MainThread:29389 [wandb_init.py:init():565] updated telemetry
2022-03-31 18:47:07,516 INFO    MainThread:29389 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-31 18:47:08,289 INFO    MainThread:29389 [wandb_run.py:_on_init():1759] communicating current version
2022-03-31 18:47:08,362 INFO    MainThread:29389 [wandb_run.py:_on_init():1763] got version response 
2022-03-31 18:47:08,362 INFO    MainThread:29389 [wandb_init.py:init():625] starting run threads in backend
2022-03-31 18:47:10,017 INFO    MainThread:29389 [wandb_run.py:_console_start():1733] atexit reg
2022-03-31 18:47:10,018 INFO    MainThread:29389 [wandb_run.py:_redirect():1606] redirect: SettingsConsole.WRAP
2022-03-31 18:47:10,018 INFO    MainThread:29389 [wandb_run.py:_redirect():1643] Wrapping output streams.
2022-03-31 18:47:10,019 INFO    MainThread:29389 [wandb_run.py:_redirect():1667] Redirects installed.
2022-03-31 18:47:10,019 INFO    MainThread:29389 [wandb_init.py:init():664] run started, returning control to user process
2022-03-31 19:04:10,763 INFO    MainThread:29389 [wandb_run.py:_atexit_cleanup():1702] got exitcode: 0
2022-03-31 19:04:10,765 INFO    MainThread:29389 [wandb_run.py:_restore():1674] restore
